axelrod/game.py

```diff
@@ -132,4 +165,4 @@ def __eq__(self, other):
         return self.RPST() == other.RPST()
 
 
-DefaultGame = Game()
+DefaultGame = Game(3, 0, 5, 1, game_type='prisoners_dilemma')
```

Comment:
Is it wise to use a string to indicate the type rather than the typing system, or inheritance from an abstract base game?

Reply:
how do you suggest we use the typing system? i'd be up for that once we have a better idea of how to handle different games (at which point prisoners' dilemma ceases to be the `DefaultGame`) but currently it's just a Game class with certain attributes.

Reply:
okay - will leave for now